### ISSUE TITLE
Notes re: company accounts and tax return for year ending January 31, 2016

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/london-money#8]
First thing to say, these accounts are horribly overdue, but are being submitted this weekend. I will also need to re-submit the accounts to Companies House, submitted in December, because they are not consistent with the new set.

**Comment by sofer** [comment]
Point 1: I had originally included all the funds received by me via Indiegogo and the rent paid by me prior to incorporation in our accounts, but this was a mistake. A quick google search for "income received before a company is incorporated" makes it clear that I need to treat these sums as personal income. 

In any case, I have now belatedly done a reckoning of all my income and expenditure prior to incorporation and it actually comes to a net negative figure:

|Description|Total|
|----|----:|
|Indiegogo|9,787.20|
|Rent deposit|(2,200.00)|
|Rent paid|(8,800.00)|
|Desk rental at 16 Palmers Road|(513.66)|
|Other|(2,726.36)|
|Balance|(2,248.32)|

By some near-miracle, the balance is nearly identical to the rent deposit, which if all goes to plan, I (personally) should be able to retrieve when we move out. 

So, the total of £473.54 that I mistakenly transferred into our account when it was first opened (I had not done a full reckoning) can now be treated as a director's loan.

**Comment by sofer** [comment]
Point 2: Tax treatment of voluntary receipts.

*"if the character in the recipient’s hands is that of a payment made in order that the money may be used in the recipient’s business, to supplement trading or other business receipts and to enable the recipient to carry on business, or otherwise to preserve and maintain trading stability and solvency, then it will be a taxable trading receipt"*
https://www.gov.uk/hmrc-internal-manuals/business-income-manual/bim41810

If donations received *to enable the recipient to carry on business* are taxable, then donations from alumni are taxable. As is the .[Young Co-operators Prize](http://y-c-p.co.uk/). In other words, donations and awards are encouragements to *carry on business* and were given with that intention (*"paying it forward"*).

**Comment by sofer** [comment]
Point 3: rent re-charge

In November and December 2015, @izaakrogan and @besarthoxhaj (through their own company) paid the rent in order to help us through a difficult period.. We paid the rent to the landlord as usual, but we raised an invoice and described it as a "rent recharge". It was an unusual arrangement and from an accounting point of view it is a little bit untidy. 

In hindsight, it might have simplified the accounts if Bes and Izaak had just paid the rent on our behalf directly to the landlord. Alternatively, we could have just treated their payment as a loan. 

Having talked to Izaak, this is what I have now done. I have voided the sales invoices and added the income received from them Roox to the loan account. 

Once we have a charitable entity in place, they can convert that loan into a donation, if they so wish.

**Comment by sofer** [comment]
Point 4: Annual Investment Allowance.

I wasn't sure how to treat laptops, routers and printers and whether we could expense the entire cost. We can.

> You can claim capital allowances on items that you keep to use in your business - these are known as ‘plant and machinery’.
>
> In most cases you can deduct the full cost of these items from your profits before tax using annual investment allowance (AIA).

https://www.gov.uk/capital-allowances/what-you-can-claim-on

The AIA is currently £200,000, so we don't need to worry about the more complex case of writing down a capital cost over several years.

**Comment by sofer** [comment]
Point 5: Assets on the balance sheet

We don't get taxed on what's on the balance sheet. It's just something that we have to keep a record of. So, how we treat the value of fixed assets that have been 100% written down using AIA is entirely up to us. 

For convenience, I propose to write everything down 100% in the first year, so that no fixed assets remain on the balance sheet. seriously, who cares if we have a two-year old laptop or router hanging around? We have better things to do that keep track of these things in our accounts.

**Comment by sofer** [comment]
Members of the finance committee, I would be grateful if you could confirm that you are happy with the decisions outlined above.

**Comment by iteles** [comment]
@sofer, with regards to Izaak and Bes paying the rent - my understanding from our conversations was that this was **not** a loan, but rather it was overdue commission from the employment of Nikki, Jack Rans and Daniel for the Bookee project. Was this not the case? :confused:

**Comment by sofer** [comment]
@iteles It was a payment of rent in lieu of commission, but I think that may not go down well with HMRC. There is no such thing as a "rent recharge" as far as HMRC are concerned and I am not sure how I would explain it to them. HMRC may consider as rent of serviced offices or something else altogether. Possibly subject to VAT..and possibly not. 

I really don't think there is any serious danger of Izaak and Bes not finding a way to contribute their due when the time comes and the original payment was not particularly overdue either. At the time, I was holding my nose at the nature of the project and needed to pay the rent, so our too-clever-by-half workaround was what I described above--which in hindsight wasn't that clever.

**Comment by markwilliamfirth** [comment]
Might be an obvious reason why not, but might it not be simpler to classify it as [late] commission (which it was?) if the rent thing could cause confusion / problems?

**Comment by iteles** [comment]
@sofer Why don't we just change the description from rent recharge to 'F&C graduate commission' or even just 'Deskspace rent at number 14' ? Wouldn't that be simpler in terms of process and avoid us having to keep track of 'loans' which aren't really loans?

Speaking of which, does this also affect the desk rental that you, Nelson and I paid in 2015 as well and how that was handled or was that squared off?

**Comment by markwilliamfirth** [comment]
lol snap^

**Comment by sofer** [comment]
Point 1: We can do that but there is a non-recoverable cost to that solution: VAT at 20%.

Point 2: VAT does not affect the desk rental, because:
a) it pre-dates our registration for VAT; and
b) after our complete failure to collect desk rental from anybody else after May 2015 (and the general failure of the idea of charging rent), I turned our rent pre-payments into loans. Those loans remain on the books. £2,400 from me and £1,000 from you.

**Comment by sofer** [comment]
To be clear, I no longer feel safe not adding VAT to the *rent recharge*, which if my caution is well founded, means that we have two choices:

1. realise a VAT liability of £733 which will need to be paid now; or
1. keep the money received on the books as loans which can be recycled into charitable donations at some point in the future.

I have gone for option 2, but I am happy to be out-voted if anyone thinks option 1 is better (or thinks I am being too cautious about the VAT liability).

**Comment by iteles** [comment]
@sofer I think we agreed a change to the 'loan' mentioned above in our finance meeting but I can't find the minutes... Can you remember what it was?

**Comment by sofer** [comment]
We did agree it. @markwilliamfirth somehow landed himself with the job of chairing and minute taking. That doesn't seem fair, but he may know where they are.

**Comment by sofer** [comment]
The accounts were submitted on March 9th.

**Comment by markwilliamfirth** [comment]
@iteles @sofer yep it was agreed to get a contract for the rent amount for it to become a donation in future instead
